networks/kovan.config.ts

```diff
@@ -80,6 +80,10 @@ export const kovan: NetworkUserConfig = {
         factory: EthereumERC20__factory,
         name: TOKEN_NAMES.USDC,
       },
+      {
+        factory: EthereumERC20__factory,
+        name: TOKEN_NAMES.KETH,
```

Comment:
This token name is undefined. Will most likely throw an error.

Reply:
Why not WETH like other Ethereum networks?
See https://kovan.etherscan.io/token/0xd0a1e359811322d97991e03f863a0c30c2cf029c